vocab_files/attribute_concepts/outlook-zone-right-direction.ttl

```diff
@@ -0,0 +1,17 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX schema: <https://schema.org/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX tern: <https://w3id.org/tern/ontologies/tern/>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://linked.data.gov.au/def/nrm/a73f5249-542d-594f-b7d8-eb074c5405dc>
+    a skos:Concept ;
+    dcterms:source "McCallum K, Bignall J, Cox B, O’Neill S, Sparrow B. (2023) Fauna Ground Counts Module. In ‘Ecological Field Monitoring Protocols Manual using the Ecological Monitoring System Australia’. (Eds S O’Neill, K Irvine, A Tokmakoff, B Sparrow). TERN, Adelaide." ;
+    rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
+    skos:definition "The compass direction (right boundary, in degrees from north) facing the intended outlook zone from a vantage point." ;
+    skos:prefLabel "outlook zone - right boundary direction" ;
+    schema:url "https://github.com/ternaustralia/dawe-rlp-vocabs/tree/main/vocab_files/attribute_concepts/outlook-zone-right-direction.ttl"^^xsd:anyURI ;
+    tern:valueType tern:Degrees ;
```

Comment:
```suggestion
    tern:valueType tern:Float ;
```
Degree's value type is Float in our information model.

Reply:
Done